source/javascripts/meetups.js

```diff
@@ -45,7 +45,9 @@
 
   handler.buildMap(mapOptions, function() {
     if(navigator.geolocation) {
-      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap);
+      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap, function(){
```

Comment:
Yup, looks like you've done what I was proposing already :)
I'll revert my change to this part of the code from this PR so there will be no conflict when merging